.github/workflows/ci.yml

```
@@ -19,77 +19,98 @@
           fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.19.1'
       - uses: oven-sh/setup-bun@v1
         with:
-          bun-version: 1.0.4
-      - name: Install dependencies
-        run: yarn install
+          bun-version: 1.1.10
+      - run: bun install -y
         if: "${{ steps.cache-nodemodules.outputs.cache-hit != 'true' }}"
```

actionlint (1.7.7)

property "cache-nodemodules" is not defined in object type {}

(expression)